pydrive2/test/test_file.py

```diff
@@ -842,6 +842,39 @@ def test_Parallel_Insert_File_Passed_HTTP(self):
         files = self._parallel_uploader(self.FILE_UPLOAD_COUNT, 10)
         self._parallel_downloader(files, 10)
 
+    # Tests for Copy file.
+    # ====================
+
+    def test_CopyFile(self):
+        drive = GoogleDrive(self.ga)
+        file1 = drive.CreateFile()
+        filename = self.getTempFile("copytestfile")
+        content = "hello world!"
+        file1["title"] = filename
+        file1.SetContentString(content)
+        pydrive_retry(file1.Upload)
+        pydrive_retry(file1.FetchMetadata)
+
+        parent_id = file1["parents"][0]["id"]
+        parent_folder = drive.CreateFile({"id": parent_id})
+
+        file1.Copy(parent_folder, new_title="copytestfile_copy")
+
+        files = drive.ListFile({"q": f"'{parent_id}' in parents"}).GetList()
+
+        file2 = None
```

Comment:
this whole construct can be replace with something like:

`self.assertIn("copytestfile_copy", [f["title"] for f in files])`

Reply:
i also need `file2` so that i can get it's content, to compare it with `content`. Witch is the string inside `file1`.

i will add some comments to make what I'm doing a little clearer